### PR TITLE
Use shared cache by default for berkeleydb

### PR DESCRIPTION
**Pull request by mad**
When I want create two instance of berkeleydb with default configuration it lead to out of memory - default configuration 60% memory for instance

I can't proper set [MAX_MEMORY_PERCENT](https://docs.oracle.com/cd/E17277_02/html/java/com/sleepycat/je/EnvironmentConfig.html#MAX_MEMORY_PERCENT) because number of instance is unknown

Bdb has [SHARED_CACHE](https://docs.oracle.com/cd/E17277_02/html/java/com/sleepycat/je/EnvironmentConfig.html#SHARED_CACHE) it will manage shared cache by ourself

I think it would be better as default settings

Signed-off-by: Pavel Ershov <owner.mad.epa@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by farodin91** [comment]
You changed the default behaviour, so it would be good to added this as note in the documentation  https://docs.janusgraph.org/latest/upgrade.html.

**Comment by mad** [comment]
@farodin91 Ok, I added info to doc and upgrade notes

RELEASING.md has broken links for *.adoc files, I think it will be changed

From 
```
*   `docs/changelog.adoc`
*   `docs/versions.adoc`
*   `docs/upgrade.adoc`
*   `docs/xsl/common.xsl`
```

To
```
*   `docs/changelog.md`
```

May I make this fix in this PR?

**Comment by farodin91** [comment]
> @farodin91 Ok, I added info to doc and upgrade notes
> 
> RELEASING.md has broken links for *.adoc files, I think it will be changed
> 
> From
> 
> ```
> *   `docs/changelog.adoc`
> *   `docs/versions.adoc`
> *   `docs/upgrade.adoc`
> *   `docs/xsl/common.xsl`
> ```
> 
> To
> 
> ```
> *   `docs/changelog.md`
> ```
> 
> May I make this fix in this PR?

Could you open an issue? We shouldn't fix this in the same PR.

**Comment by mad** [comment]
@farodin91 sure https://github.com/JanusGraph/janusgraph/issues/1807

File: janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java [hunk 1]
```diff
@@ -110,6 +110,7 @@ private void initialize(int cachePercent) throws BackendException {
             envConfig.setAllowCreate(true);
             envConfig.setTransactional(transactional);
             envConfig.setCachePercent(cachePercent);
+            envConfig.setSharedCache(true);
```
Comment (farodin91) [inline janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java]: (nitpick) If you like, you could a config to deactivated shared cache in a certain case.

**Comment by farodin91** [comment]
@mad In the near future, we will deactivate our old cla mechanism.

File: janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java (hunk 1 above)
Comment (mad) [reply janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java]: ok

**Pull request merged**